data/lang/ui-core/en.json

```diff
@@ -1893,7 +1893,7 @@
   },
   "STATION_DOCKS": {
     "description": "Information shown in the station lobby, for total number of docking pads",
-    "message": "Our total capacity is {total_docking_pads} docking pads."
+    "message": "Your ship is docked on pad {shipbay} out of {bays} docking pads."
```

Comment:
Is the "out of bays" part necessary?

Reply:
I dont know but see the diff, how many bays there are in the station was all there was before, I only added the shipbay

Reply:
Yeah, then I realized that you replace the line, not adding another about the pads. Sorry